framework_crates/bones_bevy_renderer/src/lib.rs

```diff
@@ -1023,23 +1023,26 @@ fn extract_bones_sprites(
                 please open an issue."
                 );
             };
-            extracted_sprites.sprites.push(ExtractedSprite {
-                entity: bones_renderable_entity.0,
-                transform: {
-                    let mut t: Transform = transform.into_bevy();
-                    // Add tiny z offset to enforce a consistent z-sort
-                    t.translation.z += z_offset;
-                    z_offset += 0.00001;
-                    t.into()
+            extracted_sprites.sprites.insert(
```

Comment:
Still trying to find details about this change from `sprites: Vec<ExtractedSprite>` to an `EntityHashMap<Entity, ExtractedSprite>` and if this is the correct migration